activiti-core/activiti-spring-boot-starter/src/test/java/org/activiti/spring/boot/process/ProcessRuntimeIT.java

```diff
@@ -201,6 +201,48 @@ public void should_createNewProcessInstanceWithoutRunningIt_whenCreateIsCalled()
         assertThat(categorizeProcess.getStatus()).isEqualTo(ProcessInstance.ProcessInstanceStatus.CREATED);
     }
 
+    @Test
+    public void should_startAnAlreadyCreatedProcess_when_startCreatedProcessIsCalled() {
+
+        securityUtil.logInAs("user");
+
+        ProcessInstance categorizeProcessCreated = processRuntime.create(ProcessPayloadBuilder.start()
+            .withProcessDefinitionKey(CATEGORIZE_PROCESS)
+            .withVariable("expectedKey",
+                true)
+            .build());
+        assertThat(categorizeProcessCreated.getStatus()).isEqualTo(ProcessInstance.ProcessInstanceStatus.CREATED);
+
+        ProcessInstance categorizeProcess = processRuntime.startCreatedProcess(categorizeProcessCreated.getId());
+
+        assertThat(RuntimeTestConfiguration.completedProcesses).contains(categorizeProcess.getId());
```

Comment:
Could you add an assertion to make sure that there is a user task available on this process? You can also modify the preview test to make sure that no task is available when the process is in created status.